packages/adapter-drizzle/src/pg/index.ts

```diff
@@ -0,0 +1,244 @@
+import {
+  integer,
+  timestamp,
+  pgTable,
+  text,
+  primaryKey,
+} from "drizzle-orm/pg-core"
+import { drizzle } from "drizzle-orm/postgres-js"
+import postgres from "postgres"
+import { Adapter, AdapterAccount } from '@auth/core/adapters'
+import { and, eq } from "drizzle-orm"
+
+const queryConnection = postgres(process.env.DATABASE_URL!)
+
+export const users = pgTable("users", {
+  id: text("id").notNull().primaryKey(),
+  name: text("name"),
+  email: text("email").notNull(),
+  emailVerified: timestamp("emailVerified", { mode: "date" }),
+  image: text("image"),
+})
+
+export const accounts = pgTable(
+  "accounts",
+  {
+    userId: text("userId")
+      .notNull()
+      .references(() => users.id, { onDelete: "cascade" }),
+    type: text("type").$type<AdapterAccount["type"]>().notNull(),
+    provider: text("provider").notNull(),
+    providerAccountId: text("providerAccountId").notNull(),
+    refresh_token: text("refresh_token"),
+    access_token: text("access_token"),
+    expires_at: integer("expires_at"),
```

Comment:
why not use the native PG date column? Let the database do the work, not the consumer. 

https://orm.drizzle.team/docs/column-types/pg#timestamp

Reply:
Need to follow the spec: https://authjs.dev/reference/adapters#models